lib/ts/recipe/thirdparty/recipe.ts

```diff
@@ -76,6 +78,160 @@ export default class Recipe extends RecipeModule {
             if (mtRecipe !== undefined) {
                 mtRecipe.staticThirdPartyProviders = this.config.signInAndUpFeature.providers;
             }
+
+            const mfaInstance = MultiFactorAuthRecipe.getInstance();
```

Comment:
remove all the the mfa stuff from below since we will not support this.

Reply:
Done